src/vulkan/buffer_descriptor.cc

```diff
@@ -49,36 +36,6 @@ BufferDescriptor::BufferDescriptor(DescriptorType type,
 
 BufferDescriptor::~BufferDescriptor() = default;
 
```

Comment:
Do push constants not map to the buffer concept? Can we not just subclass buffer for this?